### ISSUE TITLE
[catalog client POC] POC Terria

### ISSUE THREAD

**Issue by asafMasa** [MapColonies/discrete-layer-client#22]
- [x] RTL
- [x] Translation
- [x] Maintenance
- [x] Changes in UI - add remove buttons / components - put our filters instead of current filters
- [x] Verify Terria can be used as a building block in our catalog app or similar apps like exporter

**Comment by alebinson** [comment]
- [X]  RTL ----- **Very pure, work should be performed**
![image](https://user-images.githubusercontent.com/23166717/100221396-4ef46d80-2f21-11eb-8c1e-730ed9910eba.png)
- [X]  Translation ---- **Translation mechanism implemented**
- [X]  Maintenance
 - Code ---- **Javascript React code old style**
 - JSX   ----- **Templates written mostly based on native HTML elements ```<button>/<input>``` etc.**  **No compliancy to MC components**
```  render() {
    const clearButton = (
      <button
        type="button"
        className={Styles.searchClear}
        onClick={this.clearSearch}
      >
        <Icon glyph={Icon.GLYPHS.close} />
      </button>
    );

    return (
      <form
        className={Styles.searchData}
        autoComplete="off"
        onSubmit={event => event.preventDefault()}
      >
        <label htmlFor="search" className={Styles.formLabel}>
          <Icon glyph={Icon.GLYPHS.search} />
        </label>
        <input
          id="search"
          type="text"
          name="search"
          value={this.props.searchText}
          onChange={this.handleChange}
          onFocus={this.props.onFocus}
          onKeyDown={this.onKeyDown}
          className={Styles.searchField}
          placeholder={this.props.placeholder}
          autoComplete="off"
          autoFocus={this.props.autoFocus}
        />
        {(this.props.alwaysShowClear || this.hasValue()) && clearButton}
      </form>
    );
  }
```
 - Build system ---- **Based on gulp task runner, might be nice**
- [X]  Changes in UI - add remove buttons / components - put our filters instead of current filters
   **When forked, changes might be performed due to our needs and standards**
- [X]  Verify Terria can be used as a building block in our catalog app or similar apps like exporter
    **Obviously when all needed changes are performed**

Bottom line: Think we **should not fork** TerriaJS, but we obviously can take parts for CSW communication
